src/benchmarks/micro/libraries/System.Text.Json/Utf8JsonReader/Perf.Get.cs

```diff
@@ -0,0 +1,148 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Text.Json.Tests
+{
+    [BenchmarkCategory(Categories.Libraries, Categories.JSON)]
+    public class Perf_Get
+    {
+        private static readonly byte[] _jsonSByteBytes = Encoding.UTF8.GetBytes(sbyte.MaxValue.ToString());
+        private static readonly byte[] _jsonInt16Bytes = Encoding.UTF8.GetBytes(short.MaxValue.ToString());
+        private static readonly byte[] _jsonInt32Bytes = Encoding.UTF8.GetBytes(int.MaxValue.ToString());
+        private static readonly byte[] _jsonInt64Bytes = Encoding.UTF8.GetBytes(long.MaxValue.ToString());
+
+
+        private static readonly byte[] _jsonSingleBytes = Encoding.UTF8.GetBytes(float.MaxValue.ToString("F"));
+        private static readonly byte[] _jsonDoubleBytes = Encoding.UTF8.GetBytes(double.MaxValue.ToString("F"));
```

Comment:
```suggestion
        private static readonly byte[] _jsonDoubleBytes = Encoding.UTF8.GetBytes(double.MaxValue.ToString());
```

This "F" format representation for this value is quite long, and it shows in the results for `double` compared to the other number types.

Reply:
My point was to get the perf. result in the worst case scenario, why we should not aim to that?

Reply:
If we had to pick one, I think it's more valuable to benchmark the most common scenarios. A 312 digit number would be a rare occurrence in a JSON payload.